src/Platform/Microsoft.Testing.Platform/Services/IPlatformInformation.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Globalization;
+using System.Reflection;
+
+namespace Microsoft.Testing.Platform.Services;
+
+public interface IPlatformInformation
```

Comment:
Shall we expose the other runtime info through this interface? Or as the logic is directly the one of runtime, we let framework authors doing the job?